packages/modules/backup_clouds/samba/backup_cloud.py

```diff
@@ -10,13 +12,17 @@
 
 
 def upload_backup(config: SambaBackupCloudConfiguration, backup_filename: str, backup_file: bytes) -> None:
-    conn = SMBConnection(config.smb_user, config.smb_password, 'client', 'server', use_ntlm_v2=True)
-    conn.connect(config.smb_path, 445)
-
-    with conn.openFile(backup_filename, 'w') as file:
-        file.write(backup_file)
-
-    conn.close()
+    conn = SMBConnection(config.smb_user, config.smb_password, os.uname()[1], config.smb_server, use_ntlm_v2=True)
+    log.info("SMB Verbindungsaufbau")
+    if conn.connect(config.smb_server,139) == True:
+        log.warn("SMB Verbindungsaufbau erfolgreich")
```

Comment:
`log.warn` sollte auch nur für Warnungen genutzt werden. Analog zu der Zeile 16 hier auch `log.info` verwenden.
```suggestion
        log.info("SMB Verbindungsaufbau erfolgreich")
```